### ISSUE TITLE
Review & submit validation error links

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#21252]
## Issue Description

This is an implementation of the discovery & design work from #2138.

Initial implementation was fraught with trouble since it made global changes and should have been split into smaller changes. To complete the work of #2138, when form 526 encounters a validation error, it will present a list of links inside the error alert. These links will open the accordion chapter and target the page & set it to edit mode. The scroll position & focus will update to keep the element in view. Once the editing has completed, the page will collapse and maintain focus on the edit button. The appropriate alert link will also be removed if the page validates. 

This work as done in https://github.com/department-of-veterans-affairs/vets-website/pull/11990, but it was still unwieldily, so it was further split out into

- [x] Accordion highlighting https://github.com/department-of-veterans-affairs/vets-website/pull/14465
- [ ] Adding links (pending)

<details><summary>Current state of validation errors</summary>

![](https://user-images.githubusercontent.com/136959/94954488-d4721900-04ae-11eb-8a92-792ce32520b6.png)</details>

<details><summary>Proposed validation links</summary>

![](https://user-images.githubusercontent.com/136959/73964336-9ac52000-48d7-11ea-9a84-677ff866870d.png)</details>

## Front end tasks

- Implement links, focus management and scrolling tasks
- Ensure unit tests are available. 
- Ensure your code changes are covered by E2E tests
- Run axe checks using the Chrome or Firefox browser plugin
- Test for color contrast and color blindness issues
- Zoom layouts to 400% at 1280px width
- Test with 1 or 2 screen readers
- Navigate using the keyboard only

## Acceptance Criteria

- [ ] Content review of link texts
- [ ] Unit tests pass
- [ ] E2E tests cover current code and regression
- [ ] End-to-end tests show 0 violations.
- [ ] The data returned matches API response (for given user or scenario)
- [ ] All axe checks pass
- [ ] All color contrast checks pass
- [ ] All zoom testing passes
- [ ] All keyboard checks pass 
- [ ] All screenreader checks pass
